test/elasticsearch.ts

```diff
@@ -113,6 +113,13 @@ describe("elasticsearch", () => {
 
   describe("update_replicas", () => {
     it("should make one command to put new index settings", (done) => {
+      if (!require("../config").indices.replicas) {
+        // TODO: methods in lib/elasticsearch.ts behave differently depending on settings in config.yml.
```

Comment:
yeah... I started down that path but right now config is a global singleton pulled via `require`... this is not easy to modify at test time